src/canmatrix/xls.py

```diff
@@ -309,6 +303,28 @@ def dump(db, file, **options):
     # save file
     workbook.save(file)
 
+############################ load ###############################
+
+def parse_value_name_collumn(value_name, value, signal_size, float_factory):
+    mini = maxi = offset = None
+    value_table = dict()
+    if ".." in value_name:
+        (mini, maxi) = value_name.strip().split("..", 2)
+        mini = float_factory(mini)
+        maxi = float_factory(maxi)
+        offset = mini
+
+    elif len(value_name) > 0:
+        if len(value.strip()) > 0:
+            # Value Table
+            value = int(float(value))
+            value_table[value] = value_name
+        maxi = pow(2, signal_size) - 1
+        maxi = float_factory(maxi)
+        mini = 0
+        offset = 1
```

Comment:
This seems odd having a default offset of 1, but I don't really know what this is doing so...  :|  I'll highlight it like this but if it's proper, just move on and ignore this comment.

Reply:
you are right here, I thought about factor when I initialized offset with 1.